go-controller/pkg/cni/helper_linux.go

```diff
@@ -315,6 +323,20 @@ func (pr *PodRequest) ConfigureInterface(namespace string, podName string, ifInf
 		}
 	}
 
+	err = netns.Do(func(hostNS ns.NetNS) error {
+		if _, err := os.Stat("/proc/sys/net/ipv6/conf/all/dad_transmits"); !os.IsNotExist(err) {
+			err = setSysctl("net/ipv6/conf/all/dad_transmits", 0)
+			if err != nil {
+				return fmt.Errorf("failed to disable IPv6 DAD: %q", err)
+			}
+		}
+		return ip.SettleAddresses(contIface.Name, 10)
+	})
+	if err != nil {
+		logrus.Warningf("Failed to disable IPv6 DAD and settle IP addresses: %v", err)
```

Comment:
you already have "failed to disable IPv6 DAD" in the `fmt.Errorf` call above so this would be a pretty redundant message

Reply:
the main additional thing this would catch is if ip.SettleAddresses() failed.  I'll check this to clean it up.